clojure-ts-mode.el

```diff
@@ -160,6 +160,14 @@ current sexp."
   :safe #'booleanp
   :type 'boolean)
 
+(defcustom clojure-ts-use-metadata-for-privacy nil
```

Comment:
Probably I'd make this a bit more explicit - e.g. `...-for-defn-privacy`. (as it's the only form with the alternate syntax)